### DISCUSSION TITLE
Import of enum doesn't work(Typescript)

### DISCUSSION THREAD

**Discussion by svjatber** [nuxt/nuxt#15940] [Questions]
I use auto-import for typescript. When I try to use enums in components I get errors. Also, I can't import enum in service files and store files. 

Maybe someone faced this issue. It'll be so nice to get an answer. Thx

**Comment by cipami** [comment]
Had the same problem, when importing enum from index.ts. Changing name to enums.ts works 🤷‍♀️

**Comment by RPSoftecco** [comment]
Signing up to this problem. I also noticed that if you have one enum.ts file, you can use enums without import in your component, but in that case you can't use them in other .ts files. 
The second: thing is if you add export wihin enum.ts at least for one of the enums, you will need to add export to the rest of the enums.
The third: if you create several files with enums (let's say enums.ts and newEnums.ts) and add export only to newEnums.ts, and it is expected to use auto-import in enums.ts, this does not solve the problem at all. Because in this case, Nuxt everywhere expects the developer to use export.

**Comment by gigabites19** [comment]
Got the same problem

**Comment by TheAlexLichter** [comment]
@svjatber @cipami @RPSoftecco @gigamarr Can anyone provide a reproduction via StackBlitz oder Codesandbox please? ☺️

**Comment by kieuminhcanh** [comment]
Same. Waiting for update

**Reply by TheAlexLichter** [reply]
Please read my comment above
